modules/lvms-about-creating-lvmcluster-cr.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-creating-lvmcluster-cr_{context}"]
+= Creating an LVMCluster custom resource
```

Comment:
Hi @skrthomas 
Like you have mentioned, there are two such modules:

Creating an LVMCluster CR and Deleting an LVMCluster CR

These two modules have two or three subsections each because LVMCluster can be created by using the CLI, web console, or RHACM.

So these modules act as introductions and have common info that is applicable to all three modules. I chose to use a gerund for these modules despite these being concept modules is because these modules hold procedures. 

WDYT of this approach?

Reply:
I can see why you did what you did. However, I still think that the gerund is slightly misleading because the user still has to scroll down to see the actual steps to create an LVMCluster CR, using either CLI or Web Console. 

Since you are documenting two ways to accomplish the same goal, and this CONCEPT serves as a kind of Overview of taking either of these approaches, maybe a title that encapsulates the concepts could be "Ways to create an LVMCluster" and "Ways to delete an LVMCluster"

Reply:
@skrthomas 
Thanks for the clarification.
Updated the titles as per the suggestion.

Reply:
/lgtm thanks @sr1kar99